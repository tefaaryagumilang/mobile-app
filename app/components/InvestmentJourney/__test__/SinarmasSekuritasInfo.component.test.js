import React from 'react';
import renderer from 'react-test-renderer';
import SinarmasSekuritasInfo from '../SinarmasSekuritasInfo.component';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';

const store = createStore(() => ({}));
const DecoratedForm = reduxForm({form: 'testForm'})(SinarmasSekuritasInfo);

describe('SinarmasSekuritasInfo: form page', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Provider store={store}><DecoratedForm/></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
