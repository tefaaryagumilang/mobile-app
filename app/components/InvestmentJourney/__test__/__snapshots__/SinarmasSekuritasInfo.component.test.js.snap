// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SinarmasSekuritasInfo: form page renders correctly 1`] = `
<RCTScrollView
  bounces={false}
  contentContainerStyle={
    Object {
      "flexDirection": "column",
      "flexGrow": 1,
      "justifyContent": "space-between",
    }
  }
  keyboardShouldPersistTaps="handled"
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "height": "100%",
    }
  }
>
  <View>
    <View
      style={
        Array [
          Object {
            "paddingTop": 8,
          },
        ]
      }
    >
      <View
        accessibilityIgnoresInvertColors={true}
        style={
          Object {
            "height": 245,
          }
        }
      >
        <Image
          borderRadius={15}
          source="mutualfund-card.png"
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "height": 245,
                "width": undefined,
              },
              undefined,
            ]
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "transparent",
              "flex": 1,
              "padding": 20,
              "paddingTop": 15,
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 12,
                  },
                  Object {
                    "color": "#FFFFFF",
                    "fontFamily": "roboto",
                    "fontWeight": "300",
                    "paddingTop": 30,
                  },
                ]
              }
            >
              Jumlah Investasi
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 16,
                  },
                  Object {
                    "alignItems": "center",
                    "color": "#FFFFFF",
                    "flex": 1,
                    "flexDirection": "row",
                    "flexWrap": "wrap",
                    "fontFamily": "roboto",
                    "fontWeight": "500",
                    "justifyContent": "space-between",
                    "marginBottom": 17,
                    "paddingHorizontal": 10,
                  },
                ]
              }
            >
              
            </Text>
          </View>
          <View>
            <Text
              style={
                Object {
                  "color": "#FFFFFF",
                  "fontFamily": "roboto",
                  "fontSize": 22,
                  "fontWeight": "500",
                  "marginBottom": 17,
                }
              }
            >
              
               
              0
            </Text>
          </View>
          <View
            style={
              Object {
                "backgroundColor": "#ECECEC",
                "height": 1,
                "opacity": 0.2,
              }
            }
          />
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={Object {}}
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 12,
                    },
                    Object {
                      "color": "#FFFFFF",
                      "fontFamily": "roboto",
                      "fontWeight": "300",
                      "paddingLeft": 5,
                      "paddingTop": 10,
                    },
                  ]
                }
              >
                NAB per unit (
                13/06/2023
                )
              </Text>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 12,
                    },
                    Object {
                      "color": "#FFFFFF",
                      "fontFamily": "roboto",
                      "paddingLeft": 5,
                    },
                  ]
                }
              >
                
                 
                0
              </Text>
            </View>
            <View
              style={
                Object {
                  "borderColor": "#ECECEC",
                  "borderRightWidth": 1,
                  "height": 40,
                  "marginHorizontal": 20,
                  "marginVertical": 10,
                  "opacity": 0.2,
                }
              }
            />
            <View
              style={Object {}}
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 12,
                    },
                    Object {
                      "color": "#FFFFFF",
                      "fontFamily": "roboto",
                      "fontWeight": "300",
                      "paddingTop": 10,
                    },
                  ]
                }
              >
                Jumlah Unit
              </Text>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 12,
                    },
                    Object {
                      "color": "#FFFFFF",
                      "fontFamily": "roboto",
                    },
                  ]
                }
              >
                0
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "bottom": 15,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "marginHorizontal": 50,
                "marginLeft": 45,
                "position": "absolute",
              }
            }
          >
            <View>
              <View
                accessible={true}
                clickable={true}
                isTVSelectable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "opacity": 1,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": undefined,
                        "fontSize": 25,
                      },
                      Object {
                        "alignItems": "center",
                        "color": "#FFFFFF",
                        "paddingHorizontal": 40,
                      },
                      Object {
                        "fontFamily": "simas",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  î©­
                </Text>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontSize": 12,
                      },
                      Object {
                        "color": "#FFFFFF",
                        "fontFamily": "roboto",
                        "fontWeight": "300",
                        "paddingHorizontal": 20,
                      },
                    ]
                  }
                >
                  Pembelian
                </Text>
              </View>
            </View>
            <View>
              <View
                accessible={true}
                clickable={true}
                isTVSelectable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "opacity": 1,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": undefined,
                        "fontSize": 25,
                      },
                      Object {
                        "alignItems": "center",
                        "color": "#FFFFFF",
                        "paddingHorizontal": 40,
                      },
                      Object {
                        "fontFamily": "simas",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  î©¬
                </Text>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontSize": 12,
                      },
                      Object {
                        "color": "#FFFFFF",
                        "fontFamily": "roboto",
                        "fontWeight": "300",
                        "paddingHorizontal": 20,
                      },
                    ]
                  }
                >
                  Penjualan
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginBottom": 10,
            "paddingHorizontal": 20,
            "paddingTop": 20,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontSize": 16,
              },
              Object {
                "fontFamily": "roboto",
              },
            ]
          }
        >
          Tidak ada riwayat transaksi
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "bottom": 0,
          "height": 40,
          "position": "absolute",
        }
      }
    >
      <Text
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        Tanggal & Waktu Transaksi mengikuti Waktu Indonesia Barat
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
